src/audio/eq_fir/eq_fir.c

```diff
@@ -315,13 +331,16 @@ static int eq_fir_init_coef(struct comp_dev *dev, struct sof_eq_fir_config *conf
 #if defined FIR_MAX_LENGTH_BUILD_SPECIFIC
 		if (eq->length * nch > FIR_MAX_LENGTH_BUILD_SPECIFIC) {
 			comp_err(dev, "Filter length %d exceeds limitation for build.",
-				 fir[i].taps);
+				 eq->length);
```

Comment:
I think this specific part should be Fixes: 465e6d0e

Reply:
put this in a separate patch as it is not related to this .